frontend/src/css/app.sass

```diff
@@ -96,10 +96,66 @@ p, div
   @media (max-width: $breakpoint-sm-max)
```

Comment:
*arbitrary comment location*

On the single send page, when you enter an invalid value for the amount, it simply gives you a message on the form validation. On the batch send page, every time I type an invalid character it throws an error and shows a toast:

![batch-amount-errors](https://github.com/ScopeLift/umbra-protocol/assets/1479802/c5f97b4c-b099-4cc9-b1f8-5b8cb5dbf594)

Reply:
Added a `isValidAmount` checker https://github.com/ScopeLift/umbra-protocol/pull/491/commits/d8b4b648262e510547fee0b2dcf451f9841940a6